StandaloneMmPkg/Library/StandaloneMmMemLib/StandaloneMmMemLib.h

```diff
@@ -15,6 +15,14 @@
 #include <Library/BaseMemoryLib.h>
 #include <Library/DebugLib.h>
 
+typedef struct {
+  EFI_PHYSICAL_ADDRESS    Base;
+  UINT64                  Length;
+} NON_MM_MEMORY_MAP;
+
+#define NEXT_NON_MM_MEMORY_MAP(MemoryDescriptor, Size) \
+  ((NON_MM_MEMORY_MAP *)((UINT8 *)(MemoryDescriptor) + (Size)))
+

```

Comment:
This macro is only useful when Size is returned together with the MemoryDescriptor. We do not need the macro.

Reply:
Removed it